src/multi-selector.js

```diff
@@ -55,7 +55,9 @@ angular.module('cp-multi-selector')
 					if(!scope.displayName){
 						function mapName(array) {
 							array.map((item) => {
-								item.display_name = item.name;
+								if (item) {
+									item.display_name = item.name;
```

Comment:
It's worth looking into.
Though I do think we want cr-multi select to omit undefined when it is accidentally passed in `[{name: 'name'}, undefined]`